src/support_code_library_builder/index.ts

```diff
@@ -117,8 +120,11 @@ export class SupportCodeLibraryBuilder {
         () => this.beforeTestStepHookDefinitionConfigs
       ),
       defineParameterType: this.defineParameterType.bind(this),
-      defineStep,
-      Given: defineStep,
+      defineStep: deprecate(
+        this.defineStep('Given', () => this.stepDefinitionConfigs),
```

Comment:
I think I'd prefer this to have keyword: 'Unknown'

Reply:
Yeah that’s fair, will change.

Actually in the latest Gherkin there’s an enum which includes unknown and uses more abstract language, so I’ll bring that in and use it.